addons/payment/static/src/js/payment_form_mixin.js

```diff
@@ -292,7 +292,8 @@ odoo.define('payment.payment_form_mixin', require => {
         _prepareTransactionRouteParams: function (provider, paymentOptionId, flow) {
             return {
                 'payment_option_id': paymentOptionId,
-                'reference_prefix': this.txContext.referencePrefix,
+                'reference_prefix': this.txContext.referencePrefix !== undefined
```

Comment:
When does it happen that `referencePrefix` is undefined?

Reply:
I found out in the process of buying an event ticket, it doesn't have a reference.